ui/components/node-renderers/list-item.js

```diff
@@ -4,8 +4,15 @@ import React from 'react';
 import renderNode from '../../util/render-node';
 
 export default function ListItem({ docNode }) {
+  let markerType;
+  if (docNode.marker == "•") {
+    markerType = "list-style-bullet";
+  } else if (docNode.marker.match(/\d+/g) != null) {
+    markerType = "list-style-number";
+  }
```

Comment:
I think doing this sort of logic in the frontend is going to be pretty error prone. The markers for lists will vary depending on our document; sometimes they'll have a parenthesis, sometimes they'll be letters, other times they'll be triangles. If it's really critical that we distinguish between numbered lists and bulleted lists, we'll want to change that at the structural level (i.e. create different node types).